deploy/crds/sriovnetwork.openshift.io_v1_sriovnetworknodepolicy_cr.yaml

```diff
@@ -32,3 +32,22 @@ spec:
     rootDevices: ['0000:02:00.0', '0000:02:00.1']
   deviceType: netdevice
   isRdma: true
+
+---
+# Mlnx IB Nics doesn't support changing mtu
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: mlnx-ib-nics
+  namespace: sriov-network-operator
+spec:
```

Comment:
`linkType` shall be specified in this CR, right?

Reply:
yes, added